### PR TITLE
Search Functionality

### PR DESCRIPTION
**Pull request by Brewski-Dev**

### CODE REVIEW

**Comment by Brewski-Dev** [comment]
@Fermain Search functionality ready for review, the only thing i am uncertain about is the clearHTML function which i tried reproducing after seeing your video but it didnt quite work as intended thus for now im using just normal .innerHTML = ""; to clear out old posts.

File: js/login-user/loginUser.mjs [hunk 1]
```diff
@@ -24,7 +24,15 @@ async function loginUser(email, password) {
     const json = await response.json();
 
     const accessToken = json.accessToken;
```
Comment (Fermain) [inline js/login-user/loginUser.mjs:26]: There's a nice way to do this using `object destructing`:

```js
const { accessToken, ...profile } = json;
```

**Comment by Brewski-Dev** [comment]
@Fermain From here on out im starting on the create new post functionality.

**Review by Fermain — COMMENTED** [review]
It looks fine to me!

File: js/login-user/loginUser.mjs (hunk 1 above)
Comment (Brewski-Dev) [reply js/login-user/loginUser.mjs:26]: Il check it out.

**Comment by Brewski-Dev** [comment]
@Fermain Requesting some assistance with the newEntryPost commit, i think im on the right track but there is something missing, when i try to publish a new post i get back ```Uncaught TypeError: Property 'handleEvent' is not callable.```

**Pull request merged**